tests/simpleTest/persistency_test.cpp

```diff
@@ -90,6 +90,7 @@ TEST_F(PersistencyTest, PrimaryRestartVC) {
     create_and_run_replica(rp, b);
   }
   ASSERT_TRUE(client->run());
+  ASSERT_TRUE(false); 
```

Comment:
You probably didn't mean to check this line in.

Reply:
fixed